### PR TITLE
B 685 remove timer based token reload

### PR DESCRIPTION
**Pull request by oscarwroche**

### CODE REVIEW

File: frontend/src/components/ApolloWrapper/index.tsx [hunk 1]
```diff
@@ -27,6 +31,112 @@ const ApolloWrapper: React.FC<PropsWithChildren> = ({ children }) => {
...
+      });
+      return tokenSetResponse.data;
+    },
+    handleFetch: tokenSetResponse => {
+      if (setTokenSet) {
+        setTokenSet({ ...tokenSetResponse, creationDate: new Date() });
+      }
+    },
+    handleResponse: (operation, accessTokenField) => (tokenSetResponse: TokenSet) => {
+      if (tokenSetResponse.accessToken) {
```
Comment (ofux) [inline frontend/src/components/ApolloWrapper/index.tsx:53]: What happens if it's false? Maybe we should log an error in the `else`?

File: frontend/src/components/ApolloWrapper/index.tsx [hunk 2]
```diff
@@ -27,6 +31,112 @@ const ApolloWrapper: React.FC<PropsWithChildren> = ({ children }) => {
...
+          headers: {
+            ...headers,
+            authorization: tokenSetResponse.accessToken ? `Bearer ${tokenSetResponse.accessToken}` : "",
+          },
+        }));
+      }
+      return { [accessTokenField]: tokenSetResponse };
+    },
+    handleError: error => {
+      console.error(error);
```
Comment (ofux) [inline frontend/src/components/ApolloWrapper/index.tsx:64]: 👍 food for datadog

**Review by ofux — APPROVED** [review]

File: frontend/src/components/ApolloWrapper/index.tsx (hunk 1 above)
Comment (oscarwroche) [reply frontend/src/components/ApolloWrapper/index.tsx:53]: Yes and we should probably log out too

Comment (oscarwroche) [reply frontend/src/components/ApolloWrapper/index.tsx:53]: Log out will be automatically happen with `handleError` if an error is thrown in `handleResponse`

Comment (oscarwroche) [reply frontend/src/components/ApolloWrapper/index.tsx:53]: @ofux fixed in latest commit

**Review by Bernardstanislas — APPROVED** [review]

**Pull request merged**